examples/text_labels_and_annotations/legend_demo.py

```diff
@@ -114,7 +113,7 @@
 l = ax2.legend([(rpos, rneg), (rneg, rpos)], ['pad!=0', 'pad=0'],
                handler_map={(rpos, rneg): HandlerTuple(ndivide=None),
                             (rneg, rpos): HandlerTuple(ndivide=None, pad=0.)})
-
+fig.canvas.draw()
```

Comment:
What's the purpose of this `draw` command? If you `show` the figure, it is drawn anyways, right?

Reply:
oops prob left over from when CL was only getting called once...